src/QsCompiler/CompilationManager/CompilationManager.csproj

```diff
@@ -22,4 +21,13 @@
     <ProjectReference Include="..\Transformations\Transformations.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>TTests.Microsoft.Quantum.QsCompiler$(PublicKeySuffix)</_Parameter1>
```

Comment:
There's a typo (TTests instead of Tests), but since the build didn't break, maybe this InternalsVisibleTo isn't needed?

Reply:
You are right, it looks like solution building without it.